### PR TITLE
abduco: add pt_BR translation

### PR DESCRIPTION
**Pull request by FvFurquim**
- [ X ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ X ] The page(s) have at most 8 examples.
- [ X ] The page description(s) have links to documentation or a homepage.
- [ X ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ X ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.pt_BR/common/abduco.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# abduco
+
+> Gerenciador de sessao no terminal.
+> Mais informacoes: <http://www.brain-dump.org/projects/abduco/>.
```
Comment (kbdharun) [inline pages.pt_BR/common/abduco.md:4]:
```suggestion
> Mais informações: <http://www.brain-dump.org/projects/abduco/>.
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Translate command arguments like `{{name}}` and `{{bash}}` too**

File: pages.pt_BR/common/abduco.md (hunk 1 above)
Comment (FvFurquim) [reply pages.pt_BR/common/abduco.md:4]: I avoided using special characters to prevent strange messages in terminal. I'll fix it!

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

File: pages.pt_BR/common/abduco.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# abduco
+
+> Gerenciador de sessão no terminal.
+> Mais informações: <http://www.brain-dump.org/projects/abduco/>.
+
+- Lista sessoes:
```
Comment (navarroaxel) [inline pages.pt_BR/common/abduco.md:6]: Did we miss a tilde here?
```suggestion
- Lista sessões:
```

File: pages.pt_BR/common/abduco.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`abduco`
+
+- Anexa à uma sessão, criando-a se não existir:
+
+`abduco -A {{name}} {{bash}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/abduco.md:12]:
```suggestion
`abduco -A {{nome}} {{bash}}`
```

File: pages.pt_BR/common/abduco.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Anexa à uma sessão com `dvtm`, criando-a se não existir:
+
+`abduco -A {{name}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/abduco.md:16]:
```suggestion
`abduco -A {{nome}}`
```

File: pages.pt_BR/common/abduco.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Desanexa de uma sessão:
+
+`Ctrl + \`
+
+- Anexa à uma sessão no modo read-only (somente leitura):
+
+`abduco -Ar {{name}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/abduco.md:24]: Please translate the placeholders for get this easier for non-English speakers.
```suggestion
`abduco -Ar {{nome}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**